### ISSUE TITLE
Make this project workable on multiple architectures

### ISSUE THREAD

**Issue by imrehg** [balena-io-projects/balena-stress-test#1]
Currently, it works on armv7hf only, it seems.

**Comment by pcgeek86** [comment]
@imrehg Good idea @imrehg. This is probably doable with our Dockerfile templates. What are two other hardware platforms we should focus on supporting at first?

https://www.balena.io/docs/learn/develop/dockerfile/#dockerfile-templates

**Comment by imrehg** [comment]
Hey @pcgeek86 see this project https://github.com/balena-io-projects/multicontainer-getting-started where in the `data` or `frontend` project you can see how a single template can support all the architectures, or the `haproxy` one where the architecture specific `Dockerfile.arch` files are demoed. And feel free to ask me or @shaunmulligan for more details (hope you don't mind that I volunteered you, Shaun:)

Also, there's also https://github.com/balena-io-playground for experimental projects, in case you haven't seen it. Not suggesting that this needs to be moved there, just highlighting that there's that org as well.

**Comment by pcgeek86** [comment]
@imrehg Cool, I was mainly asking which other platforms you think I should support on this project at first. Right now I only have Raspberry Pi 3 B+ hardware, so I'd need to obtain other hardware to test on. What are the top two hardware devices that you think would be worth targeting in the short term?
